.github/workflows/ci.yml

```diff
@@ -110,7 +110,7 @@ jobs:
         brew install zstd || brew upgrade zstd
         brew install lz4 || brew upgrade lz4
         brew install openssl@1.1 || brew upgrade openssl@1.1
-        brew install Caskroom/cask/osxfuse || brew upgrade Caskroom/cask/osxfuse  # Required for Python llfuse module
+        brew install homebrew/cask/osxfuse || brew upgrade homebrew/cask/osxfuse  # Required for Python llfuse module
 
```

Comment:
so caskroom is old style (not working any more?) and homebrew/cask is new style?

Reply:
https://github.com/borgbackup/borg/pull/5537/checks?check_run_id=1491240535
Otherwise it breaks

Reply:
ok